protocol/testing/e2e/gov/bridge_test.go

```diff
@@ -168,15 +224,15 @@ func TestUpdateSafetyParams(t *testing.T) {
 			},
 			expectedProposalStatus: govtypesv1.ProposalStatus_PROPOSAL_STATUS_PASSED,
 		},
-		"Fail: invalid authority": {
+		"Failure: invalid authority": {
 			msg: &bridgetypes.MsgUpdateSafetyParams{
 				Authority: constants.AliceAccAddress.String(),
```

Comment:
Could you make an `InvalidBridgeAuthority` constant at the top of the file and use this everywhere invalid authorities are tested within this file?